modules/apps/27-interchain-accounts/keeper/grpc_query_test.go

```diff
@@ -0,0 +1,80 @@
+package keeper_test
+
+import (
+	"fmt"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	authtypes "github.com/cosmos/cosmos-sdk/x/auth/types"
+
+	"github.com/cosmos/ibc-go/modules/apps/27-interchain-accounts/types"
+)
+
+func (suite *KeeperTestSuite) TestQueryInterchainAccountAddress() {
+	var (
+		req     *types.QueryInterchainAccountAddressRequest
+		expAddr string
+	)
+
+	testCases := []struct {
+		msg      string
+		malleate func()
+		expPass  bool
+	}{
+		{
+			"empty request",
+			func() {
+				req = &types.QueryInterchainAccountAddressRequest{}
```

Comment:
This testcase is essentially redundant but also causes no harm as `req` is the same as below in "invalid counterparty portID" due to protobuf defaulting to empty string. 

Using `req = nil` here causes a `panic: invalid memory address or nil pointer dereference` as protobuf attempts to marshal the nil request.

Can be removed if anyone feels it should. Thoughts?

Reply:
I think it's fine to leave in.

Reply:
Might be good to update invalid counterparty portID to be empty with spaces and add `strings.TrimSpace` to the grpc.go code?

Reply:
Agree with this, I think it's a good call. Can handle this in a PR tomorrow